doc/nrf/libraries/bluetooth_services/mesh/sensor_cli.rst

```diff
@@ -144,7 +144,7 @@ mesh models sensor series-entry get <SensorID> <Column>
    Get the value of a column for a sensor on the server.
 
    * ``SensorID`` - Selects the sensor for which to get the entry value.
-   * ``Column`` - Start value of the column for which to get the entry value.
+   * ``Column`` - For three channel sensors: Start value of the column for which to get the entry value. For other sensors: Index of the column for which to get the entry value.
```

Comment:
```suggestion
   * ``Column`` - For the three channel sensors: Start value of the column for which to get the entry value. For other sensors: Index of the column for which to get the entry value.
```

Reply:
Really? "The three channel sensors" is not a defined set of anything so "the" feels out of place. I'd rather do "For sensors with three channels"

Reply:
Aaaah, now I understand :) Three-channel sensors! I thought channel sensors as a type of sensor and 3 of them LOL.
It makes sense.
Either "For three-channel sensors" or "For sensors with three channels" then :)